### DISCUSSION TITLE
13.5.2 mino typo

### DISCUSSION THREAD

**Discussion by MokhtarC** [Cinderella-Man/hands-on-elixir-and-otp-cryptocurrency-trading-bot#24] [General]
![image](https://user-images.githubusercontent.com/24864615/170518487-87607d9e-a160-472e-9bfb-ed59a6f8b3eb.png)

**Comment by Cinderella-Man** [comment]
Hello Mokhtar :wave: 

Thank you for pointing this out - I've updated the book and released version 0.4.0 (I forgot to do that before).

**Reply by MokhtarC** [reply]
So quick! :)

Thank you Kamil
